doc/man3/OSSL_CMP_ITAV_new_caCerts.pod

```diff
@@ -59,7 +59,8 @@ If I<newWithOld> is not NULL, it assigns to I<*newWithOld> the internal pointer
 to the certificate contained in the newWithOld infoValue sub-field of I<itav>.
 If I<oldWithNew> is not NULL, it assigns to I<*oldWithNew> the internal pointer
 to the certificate contained in the oldWithNew infoValue sub-field of I<itav>.
-Each of these pointers will be NULL if the respective sub-field is not set.
+Each of these pointers will be set to NULL if update of root CA certificate is
+not included.
```

Comment:
```suggestion
Each of these pointers will be set to NULL if no root CA certificate update 
is present or the respective sub-field is not included.
```

Reply:
Also the doc of `OSSL_CMP_ITAV_new_rootCaKeyUpdate()` needs to be extended:
An RootCaKeyUpdateContent structure is included only if *newWithNew* is not NULL.

This is what I actually meant in my yesterday's comment.

Reply:
done